tests/test_rnea.py

```diff
@@ -0,0 +1,75 @@
+import os
+import pytest
+import optas
+import pathlib
+import numpy as np
+import scipy.linalg as linalg
+import urdf_parser_py.urdf as urdf
+from scipy.spatial.transform import Rotation as Rot
+
+# from test_models import NUM_RANDOM, isclose
+
+import pybullet as pb
+
+
+def isclose(A: np.ndarray, B: np.ndarray):
+    return np.isclose(A, B, atol=8.0e-2).all()
+
+
+NUM_RANDOM = 100
+
+
+class TestRnea:
```

Comment:
Please move this class to `tests/test_models.py`. No need for the additional script.